packages/react/src/field/InfoIcon.tsx

```diff
@@ -0,0 +1,14 @@
+export const InfoIcon = (
```

Comment:
let's name this `IconInfo` - that way when you type `Icon...` your editor will start to autocomplete all available icons in the project